sdk/anomalydetector/azure-ai-anomalydetector/CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Release History
 
-## 3.0.0b4 (Unreleased)
+## 3.0.0b4 (2021-12-20)
```

Comment:
Your release date will need to be the date it is published to PyPI. It can't be in the past.